fiddler/datadog_checks/fiddler/check.py

```diff
@@ -0,0 +1,152 @@
+# import datetime
+# import json
+# from requests.exceptions import Timeout
+
+# import yaml
+import time
+
+import fiddler as fdl
+
+from datadog_checks.base import AgentCheck
+
+# from typing import Any
+
+
+# We will get one hours worth data in chunks and process them.
+bin_size = 3600
+
+# metrics_list = ['accuracy', 'histogram_drift', 'output_average', 'feature_average', 'traffic_count']
+metrics_list = ['accuracy']
+tags = list()
+
+# Connection parameters for Fiddler. Currently hard coded to get some testing done. Will be parameterizing this later.
+URL = 'https://demo.trial.fiddler.ai'
+ORG_ID = 'demo'
+AUTH_TOKEN = 'K4ph7ORDcIO2xVIEA6KxL1o1zHjZockgurhCOZOUSVs'
+
+client = fdl.FiddlerApi(url=URL, org_id=ORG_ID, auth_token=AUTH_TOKEN)
```

Comment:
You could initialise your client in the `__init__` function in your `FiddlerCheck` class

Reply:
Yes, bought this into __init__.